packages/daemon/src/boot/database.ts

```diff
@@ -30,6 +32,11 @@ export default class DB {
       logging: this.logger
     });
     await sequelize.sync();
+    await Promise.all([
+      pluginModel(sequelize),
+      jobModel(sequelize),
+      pipelineModel(sequelize)
```

Comment:
Shall we load and initialize these models dynamically?

Reply:
For such a basic component, the models doesn't seem necessary to load dynamically.